charts/swoop-bundle/README.md

```diff
@@ -43,14 +43,14 @@ To install SWOOP components:
 Once the chart has been deployed, you should see at least 3 deployments: postgres, minio and swoop-api.
 <br></br>
 <p align="center">
-  <img src="../../images/swoop-api-deployment-services.png" alt="SWOOP Deployment" width="1776">
+  <img src="../.../../images/swoop-api-deployment-services.png" alt="SWOOP Deployment" width="1776">
```

Comment:
is this path correct? I'm not seeing the image load on the readme

Reply:
fixed